src/overlays/actors/ovl_En_Akindonuts/z_en_akindonuts.c

```diff
@@ -183,7 +183,7 @@ void func_80BECC7C(EnAkindonuts* this, PlayState* play) {
 }
 
 s32 func_80BECD10(EnAkindonuts* this, Path* path, s32 arg2) {
```

Comment:
Since you are updating these functions, would you mind standardizing them (you don't have to if you don't want to). Looks like we have two documented variants (`HasReachedPoint` like in `sub_s`, and just `ReachedPoint` like in `EnJg`)

Reply:
At the risk of making this PR too large, sure. It's done :)